test/sampling.jl

```diff
@@ -170,6 +170,13 @@ check_sample_norep(a, (3, 12), 0; ordered=false)
 a = sample(3:12, 5; replace=false, ordered=true)
 check_sample_norep(a, (3, 12), 0; ordered=true)
 
+# tests of multidimensional sampling
+
+a = sample(1:4, (2, 2); replace=false)
```

Comment:
Now you can use a longer range.

Reply:
I think the other test is redundant, should I remove it?

Reply:
As you wish, but it doesn't really hurt.

Reply:
ok, i'll leave it

Reply:
thanks for your guidance